spec/spec_helper.rb

```diff
@@ -18,6 +18,8 @@
 require 'support/pg_ast_contains'
 require 'support/visitors'
 
+require 'rails_helper' if Gem.loaded_specs.key?('rspec-rails')
```

Comment:
Moving this from the spec, otherwise the specs become order dependent. The other specs need the Railtie to be loaded before running, otherwise the middleware doesn't work.